lib/screens/authentication/import_key/interactor/import_key_bloc.dart

```diff
@@ -31,6 +32,7 @@ class ImportKeyBloc extends Bloc<ImportKeyEvent, ImportKeyState> {
     } else if (event is AccountSelected) {
       _authenticationBloc.add(OnImportAccount(account: event.account, privateKey: state.privateKey.toString()));
     } else if (event is OnPrivateKeyChange) {
+      settingsStorage.inRecoveryMode = false;
```

Comment:
pls dont call settings directly. 
Use useCases. 

And i think we have a method to chancel the recovery mode in the settingsStorage,

Reply:
Yes...